encodings/fastlanes/src/bitpacking/mod.rs

```diff
@@ -80,6 +80,10 @@ impl BitPackedArray {
                     parray.len()
                 )
             }
+
+            if SparseArray::try_from(parray)?.indices().is_empty() {
+                vortex_bail!("cannot construct BitPackedArray using patches without indices");
+            }
```

Comment:
We depend pretty deeply on this being a SparseArray so I figured it should be checked in the constructor, along with the non-empty indices when present thing

Reply:
I think this is fine for now, we will want to change it eventually but then we can find all those places